### ISSUE TITLE
feat: `Message.text`

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2283]
A convenient way to represent the prominent text part of the message.

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.23.8](https://github.com/open-wa/wa-automate-nodejs/compare/4.23.7...4.23.8) (2021-11-11)

- remove useless installation instruction and fixes onButton [`#2266`](https://github.com/open-wa/wa-automate-nodejs/pull/2266)
- Release 4.23.8 [`bf5c3be`](https://github.com/open-wa/wa-automate-nodejs/commit/bf5c3be427c367eb9228d3e3d24de3dabe75c727)
- 📝  docs: Add how to for MD in readme #2276 [`ce57050`](https://github.com/open-wa/wa-automate-nodejs/commit/ce57050e3a73f5f479df695ca6ed12ad64c2e7e5)
- 🏷️  types: added `Message.text` #2283 [`6acefe9`](https://github.com/open-wa/wa-automate-nodejs/commit/6acefe9a0433a77544044429a5eede2ec706198b)
- 🔊  logs: added logs for MD detection where `multiDevice` is not `true` [`b96a190`](https://github.com/open-wa/wa-automate-nodejs/commit/b96a190846f0016116b6e62701d0824b441dc276)
- 🥚  patches update [`f3e34c6`](https://github.com/open-wa/wa-automate-nodejs/commit/f3e34c6d54c0cbf4869c2190224762e41589f662)
- 🔊  logs: now logs if session is MD [`0f34d65`](https://github.com/open-wa/wa-automate-nodejs/commit/0f34d654e6ae37cd276f2a0cc312bf159ed63b69)
- 🥚  patches update [`48d04c8`](https://github.com/open-wa/wa-automate-nodejs/commit/48d04c8652a8d0467a41cc027d69c29f4b74759f)
- 🥚 patches update [`2aed77b`](https://github.com/open-wa/wa-automate-nodejs/commit/2aed77bd5385f399294715f7997c0baf29fec1c0)
- 🥚  patches update [`09240b3`](https://github.com/open-wa/wa-automate-nodejs/commit/09240b3b317175c9cee2651ebe166d6122fce2a9)
- 🥚  patches update [`931f570`](https://github.com/open-wa/wa-automate-nodejs/commit/931f570755288253ca713e0ebf4d1b8585a68463)
- 🥚 patched update [`4b2b437`](https://github.com/open-wa/wa-automate-nodejs/commit/4b2b4379e1d5207cebe575ee4dbe212ef07d1492)
- 🥚  patches update [`6828773`](https://github.com/open-wa/wa-automate-nodejs/commit/6828773ef873480f5a632f44393bee3944c634c6)
- 🥚 patches update [`0ba7d68`](https://github.com/open-wa/wa-automate-nodejs/commit/0ba7d685066956949a6543e194d85845f8361649)
- 🥚  patches update [`d71820f`](https://github.com/open-wa/wa-automate-nodejs/commit/d71820f38c558a41d27070e06e00d94f636d1eaf)
- 🥚 patches update [`9a30d4a`](https://github.com/open-wa/wa-automate-nodejs/commit/9a30d4a2085cd65fe47fdadca6105fa0b9c690b3)
- updated types-only package [`63e0d5b`](https://github.com/open-wa/wa-automate-nodejs/commit/63e0d5bfcdce406ea923b5679cf5d266eadbd12a)
